docs/data/material/getting-started/installation/installation.md

```diff
@@ -6,24 +6,18 @@
 
 Run one of the following commands to add Material UI to your project:
 
-:::info
-The `next` tag is used to download the latest <b>pre-release</b>, v6 version. Remove it to get the current stable version.
-:::
-
-<!-- #default-branch-switch -->
```

Comment:
Let's keep this comment (Same as https://github.com/mui/material-ui/pull/43454/files#r1731548827)